prow/gerrit/adapter/adapter.go

```diff
@@ -250,46 +304,17 @@ func (c *Controller) Sync() {
 			return
 		}
 
+		var wg sync.WaitGroup
+		changeChan := make(chan Change)
+		for i := 1; i < c.workerPoolSize; i++ {
```

Comment:
This will create a number of threads every minute, these worker thread should be after line 341

Reply:
actually I was wrong, the workers threads from each run exit when changeChan is closed